src/app/features/app-news/app-news.page.ts

```diff
@@ -29,16 +30,16 @@ export class AppNewsPage implements OnInit {
     private device: Device,
     private inAppBrowser: InAppBrowser,
     private newsProvider: NewsProvider,
-    private storageProvider: StorageProvider,
+    private storageService: StorageService,
     private toastController: ToastController,
   ) { }
 
   ngOnInit() {
     this.uuid = environment.production ? this.device.uuid : 'test';
-    this.storageProvider.getAppNews(items => {
-      this.items = items || [];
-      this.getNews().subscribe();
-    });
+
+    this.storageService.getValue<NewsItemApp[]>(StorageKey.CacheAppNews)
+      .then(items => this.items = items || [])
```

Comment:
Find a solution for having to check each time whether the value exists. Maybe add an optional parameter for a default value?